src/App.css

```diff
@@ -0,0 +1,35 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
Some of these styles can probably be removed

Reply:
Agreed, but we can also wait until the next PR - where we remove the material-ui stuff.

Reply:
Waiting until the next PR to remove